src/Model/Ticket.java

```diff
@@ -1,81 +1,73 @@
 package Model;
 
+import Model.Enums.ConcertHall;
+import Model.Enums.StadiumSector;
+import View.Printable;
+
 import java.math.BigDecimal;
 import java.sql.Timestamp;
 import java.text.DecimalFormat;
-import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.time.LocalDateTime;
-import java.time.chrono.ChronoLocalDateTime;
 import java.time.format.DateTimeFormatter;
-import java.time.format.DateTimeFormatterBuilder;
-import java.time.format.FormatStyle;
 
-public class Ticket {
-    private static int ticketsCounter = 1;
-    private final LocalDateTime ticketCreationTime = LocalDateTime.now();
+public class Ticket implements ID, Printable {
     static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd 'at' hh:mm a");
 
     private static final SimpleDateFormat TIME_FORMAT = new SimpleDateFormat("HH:mm");
     private static final DecimalFormat FORMATTER = new DecimalFormat("$#,##0.00");
 
-    private int ticketId = 0;
     private ConcertHall concertHall;
     private int eventCode;
     private Timestamp eventTime;
     private boolean isPromo;
     private StadiumSector stadiumSector;
+
+    //@Min(value = 0, message = "Invalid backpack weight")
     private double backpackAllowedWeight;
+
+   // @Min(value = 0, message = "Invalid price")
     private BigDecimal price;
 
-    public int getTicketId() {
-        return ticketId;
+    public Ticket() {
+        this(ConcertHall.NOT_SPECIFIED, 3, null, false, StadiumSector.NOT_SPECIFIED, 0, 0); // How to create a Ticket without Event Time, not using "null"??
```

Comment:
start date of Timestamp is 1970-01-01 01:00:00.0
so, you can use it as default "unspecified" value (new Timestamp(0))

Reply:
Fixed. Thank you for advice!

Reply:
can't find the solution into code